src/app/about-us/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+import CallToAction from "@/components/about-us/CallToAction";
+import Hero from "@/components/about-us/Hero";
+import Stories from "@/components/about-us/Stories";
+import Team from "@/components/about-us/Team";
+
+export default function AboutUs() {
+  return (
+    <div className="flex flex-col items-center gap-y-defaultMob lg:gap-y-default bg-bg2 bg-no-repeat">
```

Comment:
It looks like there's duplicate backgrounds? If you take out `bg-bg2` there's still a background

Reply:
oh, i remember copying this from other pages.....